### ISSUE TITLE
how to load local diffusers model

### ISSUE THREAD

**Issue by sd2530615** [huanngzh/ComfyUI-MVAdapter#5]
due to my network issues, i have to download difussers model manually to my disk, is there anyway that i can load them?

**Comment by huanngzh** [comment]
Hi, we recommend running the following command in the terminal to speed up downloading.
```Bash
export HF_ENDPOINT=https://hf-mirror.com
```

In addition, we are working on supporting loading local files, which will be released soon.

**Comment by 1blackbar** [comment]
yeah using our own safetensors from checkpoints folder is really a must , its strange you guys dont already have that

**Comment by huanngzh** [comment]
> yeah using our own safetensors from checkpoints folder is really a must + loras folder , its strange you guys dont already have that

Hi, it is recommended to use `LDM Pipeline Loader`, which supports loading checkpoints from local folder. Btw, lora has been supported. Hope you enjoy it!
